lineman/app/models/base_endpoints_interface.coffee

```diff
@@ -0,0 +1,22 @@
+angular.module('loomioApp').factory 'BaseEndpointsInterface', (RestfulClient) ->
```

Comment:
Oh, i remember,  because Invitable is an interface wrapper around groups, people, email addresses, which are all stored differently in the backend.

My preference would be to highlight that this is just building upon the restfulclient (I'm not quite sure why you said yuck before) and call it an InvitablesClient, and keep it out of the Records stuff.